hack/install_copywrite.sh

```diff
@@ -41,6 +41,7 @@ pushd $tempdir > /dev/null
 popd > /dev/null
 
 rm -f "${dest_file%/*}"/copywrite*
+mkdir bin/
```

Comment:
I think this would fail the second time it was run without `-p`?

```suggestion
mkdir -p bin/
```

Reply:
Yup, my bad! I've applied your suggestion.